### ISSUE TITLE
Capture build time information

### ISSUE THREAD

**Issue by CJ-Wright** [conda-forge/conda-smithy#789]
This would be super helpful for computing how long rebuilds take.

**Comment by jakirkham** [comment]
So `conda-build` already prints this. What would this be adding?

**Comment by CJ-Wright** [comment]
We should centralize and store that data somewhere so we can query against it. Otherwise it is difficult to use the data.

**Comment by isuruf** [comment]
Can you open an issue in conda-build to store this in the package info folder?

**Comment by CJ-Wright** [comment]
@isuruf what is the best way to get that data somewhere else once it is saved?

**Comment by mingwandroid** [comment]
We should not put such details on the packages IMHO because reproducible builds are more important than this.

**Comment by CJ-Wright** [comment]
Would it be possible to send the file somewhere and then delete it before the package is uploaded? That way we get the data and the builds look the same.

**Comment by mingwandroid** [comment]
Good idea. I'd like build logs to also be saved somewhere different for the same reproducibility reasons, maybe it could all be combined in an auxiliary tarball? @msarahan any thoughts?

**Comment by scopatz** [comment]
And this is why we need a REST interface for conda-forge / bot :).

**Comment by dougalsutherland** [comment]
Another interesting thing to store would be the commit that triggered a given build, which could make it a little more reliable e.g. to tell which artifact corresponds to `master`.

Possibilities for where to send it:
- the [`libcflib`](https://github.com/regro/libcflib/) REST API, which then stores it...somewhere
- Maybe more available, but also might silently block our submissions or something: a Google form that ends up in a giant spreadsheet
